app/settings/index.ts

```diff
@@ -0,0 +1,14 @@
+import { Meteor } from 'meteor/meteor';
+
+import { settings as clientSettings } from './client/index';
+import { settings as serverSettings } from './server/index';
+import { ICachedSettings } from './server/CachedSettings';
+
+export let settings: ICachedSettings;
```

Comment:
I think i need to write `export let settings: ICachedSettings | Settings;`
and the do `import { Settings } from './client/lib/settings';`

help please. Do I need to import `ISettings` or something?